packages/mermaid/src/diagrams/sequence/svgDraw.js

```diff
@@ -115,6 +115,7 @@ export const drawKatex = async function (elem, textData, msgModel = null) {
       stopx = temp;
     }
 
+    // eslint-disable-next-line @typescript-eslint/restrict-plus-operands
     textElem.attr('x', Math.round(startx + Math.abs(startx - stopx) / 2 - dim.width / 2));
```

Comment:
This seems like it might be a bug to me.

Are one of these values potentially not a `number`?

Reply:
That's because msgModel has a default of null, so these values have an inferred type of `never`. 
Adding types should fix it.